server.go

```diff
@@ -223,6 +223,12 @@ func (s *server) handlePacket(pconn net.PacketConn, remoteAddr net.Addr, packet
 	session, ok := s.sessions[connID]
 	s.sessionsMutex.RUnlock()
 
+	// This is (potentially) a Client Hello.
```

Comment:
Just to be clear, this means that all packets sent in the first RTT need to fulfill this requirement? Do we do that in the client?

Reply:
We do, at least for gQUIC. #886 is the issue to track that we also do that for TLS.